docs/content/menu/index.md

```diff
@@ -70,5 +70,7 @@ headless: true
     - [Релиз]({{< ref "/docs/cd/Release.md" >}})
     - [Артефакты сборки]({{< ref "/docs/cd/Artifacts.md" >}})
     - [QApps]({{< ref "/docs/cd/QApps.md" >}})
+- Development
+    - [Работа с флакующими тестами]({{< ref "docs/development/DealingWithFlaky.md" >}})
```

Comment:
My proposal is to make one page - FlakyTests and merge into it all info about a process and @Flaky annotation.

Reply:
`FlakyTests` is more about the technical part of `@Flaky` annotation. But I wanted to make documentation that describes the process.

Reply:
Also, I think that mixing languages in one document is a bad idea.

Reply:
Agree